.gitignore

```diff
@@ -250,3 +250,14 @@ GSYMS
 GPATH
 tags
 TAGS
+
+# Files generated my cmake
+CMakeCache.txt
+CMakeFiles
+CMakeScripts
+Testing
+Makefile
```

Comment:
Are you sure about this? It's been tracked in the repo. Additionally, I don't think PyTorch supports in-source build.

Reply:
Yeah, I expressed to @seemethere in person that it was a bit strange that doing the standard setup.py instructions resulted in an in-source build; that shouldn't ever happen. And @xuhdev, you're right, it may be better to restrict Makefile ignore to specifically the top level.